src/webview/ssl-config-inputs.ts

```diff
@@ -0,0 +1,214 @@
+import { ObservableScope } from "inertial";
+import { applyBindings, html } from "./bindings/bindings";
+import { TLSConfig } from "../clients/sidecar";
+
+/** Reusable Custom HTML Element (Web Component) for SSL Advanced Config
+ * This component is used in the Direct Connection form to configure SSL settings
+ * @element ssl-config
+ * @attr {string} namespace - ensure form inputs have unique ids, distinguish kafka & schema configs
+ * @attr {TLSConfig} config - the original spec's config to be updated, if it exists
+ */
+export class SslConfig extends HTMLElement {
+  static formAssociated = true;
+  private _internals: ElementInternals;
+  constructor() {
+    super();
+    this._internals = this.attachInternals();
+  }
+
+  os = ObservableScope();
+  entries = new FormData();
+
+  identifier = this.os.signal<string>("");
+  configObj = this.os.signal<TLSConfig | undefined>(undefined);
+
+  verifyHostname = this.os.derive(() => {
+    if (this.configObj()?.verify_hostname === false) return false;
+    else return true;
+  });
+  truststorePath = this.os.derive(() => {
+    return this.configObj()?.truststore?.path;
+  });
+  truststorePassword = this.os.derive(() => {
+    return this.configObj()?.truststore?.password;
+  });
+  truststoreType = this.os.derive(() => {
+    return this.configObj()?.truststore?.type;
+  });
+  keystorePath = this.os.derive(() => {
+    return this.configObj()?.keystore?.path;
+  });
+  keystorePassword = this.os.derive(() => {
+    return this.configObj()?.keystore?.password;
+  });
+  keystoreType = this.os.derive(() => {
+    return this.configObj()?.keystore?.type;
+  });
+  keystoreKeyPassword = this.os.derive(() => {
+    return this.configObj()?.keystore?.key_password;
+  });
+
+  // Setter for message prop
+  set config(value: TLSConfig) {
+    this.configObj(value);
+  }
+  set namespace(value: string) {
+    this.identifier(value);
+  }
+
+  /** update the host form data so it contains all the changed values on submit
+   * and dispatch a bubble event to the host for other actions
+   */
+  updateValue(event: Event) {
+    const input = event.target as HTMLInputElement;
+    const name = input.name;
+    const value = input.type === "checkbox" ? input.checked : input.value;
+    this.entries.set(name, value.toString());
+    this._internals.setFormValue(this.entries);
+
+    this.dispatchEvent(
+      new CustomEvent("bubble", {
```

Comment:
Not sure if it's still a copy-paste artifact, but we can use maybe some more direct event name, e.g. `change`